### PR TITLE
MODFEE-244 Reformat code

### PR DESCRIPTION
**Pull request by Mykyta-Varenyk**

### CODE REVIEW

File: src/main/java/org/folio/rest/service/report/utils/LookupHelper.java [hunk 1]
```diff
@@ -223,7 +223,7 @@ public <T extends HasAccountInfo> Future<T> lookupRefundPayTransferFeeFineAction
     }
 
     return feeFineActionRepository.findActionsOfTypesForAccount(accountId,
-      List.of(REFUND, PAY, TRANSFER))
+        List.of(REFUND, PAY, TRANSFER))
       .map(this::sortFeeFineActionsByDate)
       .map(actions -> ctx.updateAccountContextWithActions(accountId, actions))
       .map(ctx)
```
Comment (alexanderkurash) [inline src/main/java/org/folio/rest/service/report/utils/LookupHelper.java:229]: Indentation?

File: src/main/java/org/folio/rest/service/report/context/HasAccountInfo.java [hunk 2]
```diff
@@ -9,7 +9,10 @@
 
 public interface HasAccountInfo {
   Account getAccountById(String accountId);
+
   Future<Void> updateAccountContextWithActions(String accountId, List<Feefineaction> actions);
+
   boolean isAccountContextCreated(String accountId);
+
```
Comment (alexanderkurash) [inline src/main/java/org/folio/rest/service/report/context/HasAccountInfo.java:16]: Do we always put new lines between fields in classes?

Comment (Mykyta-Varenyk) [reply src/main/java/org/folio/rest/service/report/context/HasAccountInfo.java:16]: We do have lines between methods

File: src/main/java/org/folio/rest/service/report/utils/LookupHelper.java (hunk 1 above)
Comment (Mykyta-Varenyk) [reply src/main/java/org/folio/rest/service/report/utils/LookupHelper.java:229]: Yes, this is a method parameter and is not related to the stream operations below

**Pull request merged**